src/frontend/src/catalog/system_catalog/rw_catalog/rw_materialized_views.rs

```diff
@@ -45,7 +47,7 @@ fn read_rw_materialized_views(reader: &SysCatalogReaderImpl) -> Result<Vec<RwMat
 
     Ok(schemas
         .flat_map(|schema| {
-            schema.iter_created_mvs().map(|table| RwMaterializedView {
+            schema.iter_all_mvs().map(|table| RwMaterializedView {
```

Comment:
Made another change, which is to ensure that all MVs are visible. Otherwise we can't check the creating mv's background_ddl status.

Reply:
cc @yezizp2012

Reply:
Do we need to expose the status as well? whether the mv is `Creating` or `Created`.

Reply:
True. I suppose I can add that as a field.

Reply:
Some bugs with `creating` / `created` status. Some creating MV shows up as `created`. I will do some investigation and open a separate PR for it. We can merge this first.